downstream/modules/platform/proc-operator-aap-troubleshooting.adoc

```diff
@@ -1,3 +1,14 @@
 [id="operator-aap-troubleshooting_{context}"]
 
-= Troubleshooting AnsibleAutomationPlatform issues  
\ No newline at end of file
+= Troubleshooting central configuration issues  
+
+* You can follow along in the command line while the operator is reconciling, this can be helpful for debugging.
+** Alternatively you can click into the deployment instance to see the status conditions being updated as the deployment goes on.
+* When troubleshooting you should look at the *AnsibleAutomationPlatform* instance for the main logs and then each individual component (*EDA*, *AutomationHub*, *AutomationController*) for more specific information. 
```

Comment:
Flagged in Vale for "look at" - Consider using 'examine' rather than 'look at' unless updating existing content that uses the term.